### PR TITLE
Blog post about Sharing Tokens for the 120th Sprint

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

File: _posts/development/2022-06-15-token-sharing.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Token Party!
+category: development
+---
+
+With the introduction of the workflows a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that.
```
Comment (saraycp) [inline _posts/development/2022-06-15-token-sharing.md:7]:
```suggestion
With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that. 
```

File: _posts/development/2022-06-15-token-sharing.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+layout: post
+title: Token Party!
+category: development
+---
+
+With the introduction of the workflows a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that. 
+
+# One Token To Rule Them All
+
+Let's say a group of maintainers automated the building of packages when new commits are pushed in their associated GitLab repositories. To do that, those users needed a Workflow Token, and that token is bound to a single user. The only way to trigger the workflow by more than one user was to share the user credentials.
```
Comment (saraycp) [inline _posts/development/2022-06-15-token-sharing.md:11]:
```suggestion
Let's say a group of maintainers automated the building of packages when new commits are pushed into their associated GitLab repositories. To do that, those users needed a Workflow Token, and that token is bound to a single user. The only way to trigger the workflow by more than one user was to share the user credentials.
```

File: _posts/development/2022-06-15-token-sharing.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+</figure>
+
+Click on it and you'll get to the Token User page. From there, you can add single users or whole groups, whose users will be able to work with those tokens also.
+
+<figure>
+  <img src="/images/posts/sprint_120_token_users_page.png" alt="Token Users Page" style="width: 80%">
+  <figcaption>The Token's users page</figcaption>
+</figure>
+
+As always, try it and tell us what do you think!
```
Comment (saraycp) [inline _posts/development/2022-06-15-token-sharing.md:31]:
```suggestion
As always, try it and tell us what you think!
```

File: _posts/development/2022-06-15-token-sharing.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+With the introduction of the workflows a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that. 
+
+# One Token To Rule Them All
+
+Let's say a group of maintainers automated the building of packages when new commits are pushed in their associated GitLab repositories. To do that, those users needed a Workflow Token, and that token is bound to a single user. The only way to trigger the workflow by more than one user was to share the user credentials.
+
+It works, but it's far from perfect.
+
+A better solution is to share that token with your friends. Now, you can add other users and groups as owners of a workflow token, and those users will be able to add and remove other users or groups as well. It's perfect for a team, because not only your friends can help you manage your project's tokens but they can trigger the token workflows as well.
```
Comment (saraycp) [inline _posts/development/2022-06-15-token-sharing.md:15]:
```suggestion
A better solution is to share that token with your colleagues. Now, you can add other users and groups as owners of a workflow token, and those users will be able to add and remove other users or groups as well. It's perfect for a team, because not only your colleagues can help you manage your project's tokens but they can trigger the token workflows as well.
```

**Review by saraycp — CHANGES_REQUESTED** [review]

File: _posts/development/2022-06-15-token-sharing.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Token Party!
+category: development
+---
+
+With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that.
```
[the remark this answers is not in the document]
Comment (krauselukas) [reply _posts/development/2022-06-15-token-sharing.md:7]: `We're gonna find out how to fix that` sounds like you its still WIP. I would rephrase this a bit

[the remark this answers is not in the document]
Comment (krauselukas) [reply _posts/development/2022-06-15-token-sharing.md:7]:
```suggestion
With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We've fixed that for you.
```

[the remark this answers is not in the document]
Comment (krauselukas) [reply _posts/development/2022-06-15-token-sharing.md:7]: Something like this

File: _posts/development/2022-06-15-token-sharing.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+layout: post
+title: Token Party!
+category: development
+---
+
+With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level also. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that. 
+
+# One Token To Rule Them All
+
+Let's say a group of maintainers automated the building of packages when new commits are pushed into their associated GitLab repositories. To do that, those users needed a Workflow Token, and that token is bound to a single user. The only way to trigger the workflow by more than one user was to share the user credentials.
```
Comment (krauselukas) [inline _posts/development/2022-06-15-token-sharing.md:11]:
```suggestion
Let's say a group of maintainers automated the rebuild of packages when new commits are pushed into their associated GitLab repositories. To do that, those users needed a Workflow Token, and that token is bound to a single user. The only way to trigger the workflow by more than one user was to share the user credentials.
```

File: _posts/development/2022-06-15-token-sharing.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level too. But, until now, you could not use the same token with a group of users. We're gonna find out how to fix that. 
+
+# One Token To Rule Them All
+
+Let's say a group of maintainers automated the rebuild of packages when new commits are pushed into their associated GitLab repositories. To do that, those users needed a Workflow Token, and that token is bound to a single user. The only way to trigger the workflow by more than one user was to share the user credentials.
+
+It works, but it's far from perfect.
+
+A better solution is to share that token with your colleagues. Now, you can add other users and groups as owners of a workflow token, and those users will be able to add and remove other users and groups as well. It's perfect for a team, because not only your colleagues can help you manage your project's tokens but they can trigger the token workflows as well.
```
Comment (krauselukas) [inline _posts/development/2022-06-15-token-sharing.md:15]:
```suggestion
A better solution is to share that workflow token with your colleagues. Now, you can add other users and groups as owners of a workflow token, and those users will be able to add and remove other users and groups as well. It's perfect for team collaboration, because your colleagues can help you to manage and trigger your workflows.
```

Comment (krauselukas) [reply _posts/development/2022-06-15-token-sharing.md:15]: There are a lot of `as well` phrases, just a suggestion to avoid this a bit

File: _posts/development/2022-06-15-token-sharing.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (krauselukas) [reply _posts/development/2022-06-15-token-sharing.md]: @danidoni the last comment is still valid, what do you think about changing the `We're gonna find out how to fix that` part? Not necessarily to my suggestion, but somehow to make clear its already fixed :)

File: _posts/development/2022-06-15-token-sharing.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Token Party!
+category: development
+---
+
+With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level too. But, until now, you could not use the same token with a group of users. We've fixed that for you.
```
Comment (krauselukas) [inline _posts/development/2022-06-15-token-sharing.md:7]:
```suggestion
With the introduction of the workflows, a wide range of integrations became available for individual users. Now those integrations start to get interesting at team level too. But, until now, you could not use the same workflow token with a group of users. We've fixed that for you. 
```

**Review by krauselukas — APPROVED** [review]
